services/sonarqube/sonarqube.service.js

```diff
@@ -8,15 +8,9 @@ const {
   coveragePercentage: coveragePercentageColor,
 } = require('../../lib/color-formatters')
 
-// This legacy service should be rewritten to use e.g. BaseJsonService.
-//
-// Tips for rewriting:
-// https://github.com/badges/shields/blob/master/doc/rewriting-services.md
-//
-// Do not base new services on this code.
-module.exports = class Sonarqube extends LegacyService {
+class SonarqubeCoverage extends LegacyService {
   static get category() {
-    return 'quality'
+    return 'coverate'
```

Comment:
typo here, should be `coverage`

Reply:
Huh, I'm confused why the unit tests did not detect this.